spec/factories/teams.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :team do
     sequence(:name) { |n| "Patriots#{n}" }
-    city "New England"
+    location "New England"
```

Comment:
Awesome! That is definitely a good _A-ha_ moment. A good thing to do is always run your tests before you push and especially before you merge. Maybe I'll integrate circle with this project so it will do it for us..